cfme/fixtures/pytest_selenium.py

```diff
@@ -858,9 +858,13 @@ def first_from(*locs, **kwargs):
 
     """
     assert len(locs) > 0, "You must provide at least one locator to look for!"
+    single = kwargs.pop("single", True)
     for locator in locs:
         try:
-            return element(locator, **kwargs)
+            if single:
```

Comment:
I agree. I think I will stick to good ol' "|".join in this case because I lost track of where are such locatorsfor the tabs.